### PR TITLE
TestCase class for runner to utilize per test in suite

### PR DESCRIPTION
**Pull request by clacroix12**
Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

File: ocsci/framework.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+log = logging.getLogger(__name__)
+
+
+class TestCase(object):
+    """
+        Generic test case object to contain configuration details from suite.
+        Handles test execution as well as pre and post execution steps
+        for Report Portal and Polarion.
+
+    Attributes
```
Comment (petr-balogh) [inline ocsci/framework.py]: Args:

File: ocsci/framework.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+from utility.polarion import post_to_polarion
+from utility.utils import create_unique_test_name, configure_logger, timestamp
+from .enums import TestStatus
+
+log = logging.getLogger(__name__)
+
+
+class TestCase(object):
+    """
+        Generic test case object to contain configuration details from suite.
```
Comment (petr-balogh) [inline ocsci/framework.py]: One indent left for this text block

File: ocsci/framework.py (hunk 1 above)
Comment (clacroix12) [reply ocsci/framework.py]: These are not arguments though, args are outlined in the pydoc for the `__init__`. These are object attributes. Not all are passed to the `__init__` upon instantiation.

File: ocsci/framework.py (hunk 2 above)
Comment (clacroix12) [reply ocsci/framework.py]: fixed

**Pull request merged**